iox_query/src/frontend/reorg.rs

```diff
@@ -107,16 +109,31 @@ impl ReorgPlanner {
     ///
     /// The plan looks like:
     ///
+    /// ```text
     /// (Split on Time)
     ///   (Sort on output_sort)
     ///     (Scan chunks) <-- any needed deduplication happens here
+    /// ```
+    ///
+    /// The output execution plan has `N` "output streams" (DataFusion
+    /// partitions) where `N` = `split_times.len() + 1`. The
+    /// time ranges of the streams are:
     ///
-    /// The output execution plan has n "output streams" (DataFusion partition):
     /// Stream 0: Rows that have `time` *on or before* the `split_times[0]`
-    /// Stream i (0 < i < split_times's length): Rows that have  `time` in range `(split_times[i-1], split_times[i]]`
-    /// Stream n (n = split_times.len()): Rows that have `time` *after* all the split_times and NULL rows
     ///
-    /// For example, if the input looks like:
+    /// Stream i, where 0 < i < split_times.len(): Rows that have
+    /// `time` in range `(split_times[i-1], split_times[i]]` (greater than split_times[-1]
+    /// and up to and including `split_times[i]`)
```

Comment:
I had to stare at that a second to realize you're stating the same thing twice.
Separating the repeat on its own line might help.  something like:
```suggestion
    /// Stream i, where 0 < i < split_times.len(): 
    /// Rows have: `time` in range `(split_times[i-1], split_times[i]]`,
    ///  Which is: greater than `split_times[i-1]` up to and including `split_times[i]`.
```